modules/ROOT/pages/manage-backup-restore.adoc

```diff
@@ -105,13 +122,50 @@ To restore a cluster on Runtime Fabric, create a cluster *without* running the `
 ----
 kubectl scale --replicas=0 -n rtf deployment.apps/agent
 ----
-
++
+[start =5]
++
 . Restore the cluster from the backup: 
 [source,copy]
 ----
 ./rtfctl restore <path_to_backup_file>
 ----
-+
 This process may require several minutes to complete.
 
-For Runtime Fabric, confirm the `rtfctl` binary is present in the current directory and the Kubernetes (kubectl) context is set to the cluster you are restoring to.
\ No newline at end of file
+For Runtime Fabric, confirm the `rtfctl` binary is present in the current directory and the Kubernetes (kubectl) context is set to the cluster you are restoring to.
+
+== Create a Backup and Restore for OpenShift Clusters
+
+In the following example, you have a Runtime Fabric with a set of Mule applications and configurations deployed on cluster (named `cluster-1`) inside an OpenShift cluster. To create a backup and perform a restore, follow these steps:
+
+. Run the backup command: 
++
+[source,copy]
+----
+./rtfctl backup <path_to_backup_file>
+----
++
+[start=2]
+. Shutdown `cluster-1`.
+. Remove and shutdown all Mule application deployments.
+. Remove Runtime Fabric installation.
+. Remove Runtime Fabric operator from the OpenShift cluster (optional).
+. Using `curl` hit the management plane API endpoint to regenerate the activation ID. +
```

Comment:
```suggestion
. Using `curl`, hit the management plane API endpoint to regenerate the activation ID. +
```

Reply:
I'm not sure "hit" will translate well here, although it makes sense in English. Is there another word we can use? "Access"? I don't know if that changes meaning...